typhos/cli.py

```diff
@@ -129,7 +129,18 @@
     action='store_true',
     help='Use the QDarkStyleSheet shipped with Typhos',
 )
-parser.add_argument('--stylesheet', help='Additional stylesheet options')
+parser.add_argument(
+    "--stylesheet-override", "--stylesheet",
+    help="Override all built-in stylesheets, using this stylesheet instead.",
+)
+parser.add_argument(
+    "--stylesheet-add",
+    help=(
+        "Include an additional stylesheet in the loading process. "
+        "This stylesheet will take priority over all built-in stylesheets, "
+        "but not over a template or widget's styleSheet property."
```

Comment:
Not a bug, but this argument description read to me as allowing the user to apply multiple stylesheets by invoking this argument multiple times.  If you do this, only the last `--stylesheet-add STYLESHEET` will be applied, and maybe more confusingly there's no error thrown.  

* The simple option is to just add a little bit to the help documentation notifying users that the option should only be used once. 
* A more involved path is to support multiple stylesheets added using this option, using `type='append'`, and modifying `compose_stylesheets` to take lists of stylesheets.

I like the second option, but I'd also be surprised if people were trying to set all their stylesheets through the cli when the environment variable settings exist.

Reply:
I'd lean toward the second as well, since we're touching on this code anyway.

It seems not a big ask to switch `path` to `paths` in the `compose_stylesheets` function.

Reply:
The truth is that I wouldn't have included this arg at all if I was redoing this from nothing, but since there was an existing "override the stylesheet with one path" I extended it slightly.
It's not a big ask to support multiple stylesheets from the arg here as you both describe :+1: